internal/engine/image_build_and_deployer.go

```diff
@@ -20,6 +20,26 @@ import (
 	"k8s.io/api/core/v1"
 )
 
+type imageManifest interface {
```

Comment:
hmmmm...this seems like an interface that all manifests should implement, not just imageManifests

Reply:
oh I meant to remove this, I didn't end up using it.